content/ngrx/do-not-put-everything-in-the-store.md

```diff
@@ -16,7 +16,10 @@ The NgRx core team has come up with a principle called **SHARI**, that can be us
 - Retrieved: State that needs to be retrieved with a side effect, e.g. an HTTP request
 - Impacted: State that is impacted by other components
 
+Don't over-engineer your state management! Most of the times information comes along via Ajax or WebSockets and therefore is handled on the server side. Always ask yourself when and why to put something in a client side store and keep alternatives in mind. For example use routes to reflect applied filters on a list or use a Behavior Subject in a service if you need to save some simple data like settings. Mike Ryan gave a very good talk on this topic: [You might not need NgRx](https://youtu.be/omnwu_etHTY)
```

Comment:
`BehaviorSubject` instead of Behavior Subject (better formatting :))

Reply:
Done